src/class-model.ts

```diff
@@ -49,6 +49,9 @@ export interface SnapshottedViewOptions<V, T extends IAnyClassModelType> {
   /** A function for converting the view value to a snapshot value */
   createSnapshot?: (value: V) => any;
 
+  /** A function that determines whether the view should be tracked for changes. */
+  shouldTrackChanges?: (node: Instance<T>) => boolean;
```

Comment:
Let me know if you have a better name for this!

Reply:
I think we could probs give it a name that emphasizes something a bit less "internal". Maybe something like `shouldEmitPatchOnChange`?

Reply:
`enabled:`?

Reply:
`enabled:` feels too generic imo. Users could confuse `enabled: true` to mean "hydreate the view from the snapshot" when that's not what this is used for.

I'll go with `shouldEmitPatchOnChange`.